### ISSUE TITLE
import statement from example does not work

### ISSUE THREAD

**Issue by bendavis78** [hanazuki/node-jsonnet#17]
When trying to use the import statement from the example:

```javsacript
import {Jsonnet} from '@hanazuki/node-jsonnet'; 
```
 
I get this error:
```
import {Jsonnet} from '@hanazuki/node-jsonnet';
        ^^^^^^^
SyntaxError: The requested module '@hanazuki/node-jsonnet' is expected to be of type CommonJS, which does not support named exports. CommonJS modules can be imported by importing the default export.
For example:
import pkg from '@hanazuki/node-jsonnet';
const {Jsonnet} = pkg;
```

Using nodejs v12.19.0

**Comment by hanazuki** [comment]
Thank you for your report.
The code snippet on the README is actually written in TypeScript, and the library didn't provide ESM support for Node.js yet.

I just pushed a new revision to the GitHub master branch that includes ESM support, which I expect to work with Node.js 12 and 14.
Could you try that version by `npm install git+https://github.com/hanazuki/node-jsonnet` and see it works in your settings?

**Comment by bendavis78** [comment]
Thanks, I just updated from git and ESM import now works!

**Comment by hanazuki** [comment]
Thank you!
The code is now published as v0.4.2 on the npm registry.
